src/EditorFeatures/CSharpTest/AddAccessibilityModifiers/AddAccessibilityModifiersTests.cs

```diff
@@ -20,77 +22,122 @@ public partial class AddAccessibilityModifiersTests : AbstractCSharpDiagnosticPr
         internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderAndFixer(Workspace workspace)
             => (new CSharpAddAccessibilityModifiersDiagnosticAnalyzer(), new CSharpAddAccessibilityModifiersCodeFixProvider());
```

Comment:
can we stop subclassing this base type?  and/or remove this override?  seems unnecessary since 'VerifyCS' handles this.

Reply:
Eventually, yes. We do have some tests getting inherited from it so those will need to be figured out separately.

Reply:
I have now eliminated the need to use this base type.